jwst/rscd/tests/test_rscd.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+
+from jwst.datamodels import RampModel
+from jwst.datamodels import dqflags
+from jwst.rscd.rscd_sub import correction_skip_groups
+
+
+def test_rscd_baseline_set_groupdq():
+    """
+    For a 2 integration exposure, test if the rscd code sets the groupdq flag on
+    the n groups to 'do_not_use' for the 2nd integrations and did not change the
+    groupdq flags in the 1st integration
+    """
+
+    # size of integration
+    ngroups = 10
+    xsize = 1032
+    ysize = 1024
+
+    # create the data and groupdq arrays
+    csize = (2, ngroups, ysize, xsize)
+    data = np.full(csize, 1.0)
```

Comment:
`data` will be `float64`.  That's OK but unnecessary.

Reply:
What should it be?

Reply:
Ok. I found it in the schema - it is float32.  Done.